python/tests/test_tool_definition_decorator.py

```diff
@@ -0,0 +1,153 @@
+# Copyright (c) 2024 Baidu, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+import os
+import unittest
+from appbuilder import FunctionView, function, function_parameter, function_return
+from typing import Any, Dict, List, Optional, Union
+
+@unittest.skipUnless(os.getenv("TEST_CASE", "UNKNOWN") == "CPU_PARALLEL", "")
+class TestToolDefinitionDecorator(unittest.TestCase):
+    def test_disable_docstring(self):
+        @function(description="anotated function", disable_docstring=True)
+        @function_parameter(name="param", description="a parameter", type="str", default_value="default_val")
+        @function_return(description="a result", default_value="default_result")
+        def func(param: str) -> str:
+            return param
+
+        view = func.__pf_function__
```

Comment:
__pf_function__ should change to ab_tool or sth,  cause pf represents Kaiwu's framework abbr.